packages/core-mobile/app/components/TabViewAva.tsx

```diff
@@ -90,7 +90,11 @@ const TabViewAva: TabViewAvaFC = ({
   )
 
   const navState = useMemo(() => {
-    return { index: currentIndex, routes }
+    return {
+      index:
```

Comment:
can you add a comment explaining this?

Reply:
<img width="522" alt="Screenshot 2024-09-25 at 11 03 25 AM" src="https://github.com/user-attachments/assets/af404f33-69b9-459f-ba1d-1592abf3c4bf">

I noticed when the crash happens, the currentIndex were set to random number, reset it to no bigger than the number of routes to avoid crash.